src/manifests_cert_manager.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2024 Ubuntu
+# See LICENSE file for licensing details.
+
+"""Implementation of cert-manager kubernetes manifests."""
+
+import logging
+import pickle
+from hashlib import md5
+from typing import Dict
+
+from ops.manifests import ConfigRegistry, ManifestLabel, Manifests, Patch
+
+log = logging.getLogger()
+
+
+class RemoveNamespace(Patch):
+    """Remove namespace from resources so they deploy to the charm's namespace."""
+
+    def __call__(self, obj):
+        """Remove namespace from resources."""
+        obj.metadata.namespace = None
+
+
+class CertManagerManifests(Manifests):
+    """Deployment details for cert-manager."""
+
+    def __init__(self, charm, charm_config):
+        super().__init__(
+            "cert-manager",
+            charm.model,
+            "upstream/cert-manager",
+            [
+                ManifestLabel(self),
```

Comment:
This is a nice manipulation because it labels the Manifests that they are deployed from this application. 

https://github.com/canonical/ops-lib-manifest/blob/main/ops/manifests/manipulations.py#L176